extension/src/popup/components/Header.tsx

```diff
@@ -42,12 +38,6 @@ type HeaderProps = {
 };
 
 export const Header = ({ className, ...props }: HeaderProps) => {
-  const applicationState = useSelector(applicationStateSelector);
-
-  if (applicationState === APPLICATION_STATE.APPLICATION_LOADING) {
-    return null;
-  }
```

Comment:
this was a dumb idea i tried near the beginning and was preventing views that required the Header from just including <Header /> in their render